R/preprocessing.R

```diff
@@ -0,0 +1,152 @@
+##' Fills missing data in a data set to prepare it for use within the package
+##'
+##' @description `r lifecycle::badge("experimental")`
+##'
+##' @param data Data frame with a `date` column. The other columns depend on the
+##'   model that the data are to be used, e.g. [estimate_infections()] or
+##'   [estimate_secondary()]. See the documentation there for the expected
+##'   format.
+##' @param missing Character. Options are "missing" (the default), "accumulate"
+##'   and "zero". This determines how missing dates in the data are interpreted.
+##'   If set to "missing", any missing dates in the observation data will be
+##'   interpreted as missing and skipped in the likelihood. If set to
+##'   "accumulate", modelled observations will be accumulated and added to the
+##'   next non-missing data point. This can be used to model incidence data that
+##'   is reported at less than daily intervals. If set to "accumulate", the
```

Comment:
```suggestion
##'   next non-missing data point. This can be used to model incidence data that
##'   is reported at irregular/inconsistent intervals. If set to "accumulate", the
```

Reply:
I've tried to rephrase slightly differently in an attempt to make it clearer.